homeassistant/components/lastfm/config_flow.py

```diff
@@ -0,0 +1,75 @@
+"""Config flow for LastFm."""
+from typing import Any
+
+from pylast import LastFMNetwork, WSError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_API_KEY
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_USERS, DOMAIN, LOGGER, PLACEHOLDERS
+
+
+class LastFmFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Config flow handler for LastFm."""
+
+    async def _async_setup_form(
```

Comment:
Please remove this function, and use instead a schema constant together with `add_suggested_values_to_schema`